index.js

```diff
@@ -1,17 +1,60 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
+    return this.items;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos >= this.items.length) {
+      throw new Error('OutOfBounds');
+    }
+    return this.items[pos]
+  }
 
-  max() {}
+  max() {
+    let ultimoNumero = this.items.length - 1;
+    if (this.items.length === 0) {
+      throw new Error('EmptySortedList');
+    }
+    return this.items[ultimoNumero];
+  }
 
-  min() {}
+  min() {
+    if (this.items.length === 0) {
+      throw new Error('EmptySortedList');
+    }
+    return this.items[0];
+  }
 
-  sum() {}
+  sum() {
+    let suma = 0;
+    if (this.items.length === 0) {
+      return 0;
+    }
+    else {
+      this.items.forEach(function (numero) {
+        suma += numero;
+      })
+    }
+    return suma;
```

Comment:
Fijate que esta funcion se puede reducir a:

`return this.items.reduce((num, acc) => num + acc, 0);`

Reply:
sii luego me di cuenta q podia hacerla con reduce